src/app/space/create/deployments/apps/deployment-details.component.ts

```diff
@@ -133,10 +133,10 @@ export class DeploymentDetailsComponent {
   cpuVal: number;
   cpuMax: number;
   memVal: number;
-  memUnits: string;
   memMax: number;
+  memUnits = 'MB';
   netVal: number;
-  netUnits: string;
+  netUnits = 'bytes';
```

Comment:
Even if it's obvious, I'd prefer to keep the type declarations on these properties, so `netUnits: string = 'bytes';`